lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packages": ["packages/*", "scripts", "apps/*"],
+  "packages": ["packages/*", "scripts", "apps/*", "experimentals/*"],
```

Comment:
I agree with other comments.  This should be called experiments rather than experimentals, that matches the pattern in fabric-web.

Reply:
I don't know why I typed `experimentals`. I do prefer `experiments`.